.maintain/frame-weight-template.hbs

```diff
@@ -47,6 +47,9 @@ pub trait WeightInfo {
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	{{~#each benchmarks as |benchmark|}}
+	{{~#each benchmark.comments as |comment|}}
+	// {{comment}}
+	{{~/each}}
```

Comment:
Can we adjust this to be a section that looks more like this?
```
// Storage: 
// Assets Asset (r:1 w:1)
// Assets Account (r:2 w:2)
// System Account (r:1 w:1)
```

Reply:
They are pretty similar: https://github.com/paritytech/substrate/pull/9090/files#diff-ce55fb9993890db847dac5b3e8d73fcc5844f24b51f90ed8eca964658d0460bbR379